addons/languages/coptic/pack/src/main/res/xml/coptic_scriptorium.xml

```diff
@@ -55,6 +54,6 @@
         <Key android:codes="1005"/>
         <Key android:codes="1007"/>
         <Key android:codes="11407"/>
-        <Key android:keyWidth="15%p" android:codes="-5" android:keyEdgeFlags="right" android:isRepeatable="true"/>
+        <Key android:codes="-5" android:keyEdgeFlags="right" android:isRepeatable="true"/>
```

Comment:
I'm confused what you meant by moving a key from the third row to the second; the second row seems full already?

If you want to move a key rather than dropping the shift, I would suggest making `ⳉ` (11465) a popup on `ϧ` (999), and then moving `ⲏ` (11407) up to the right edge of the second row. ⳉ has the same value as ϧ and is used only in the less common Akhmimic dialect, so it's probably the easiest to lose. ⲏ is quite common, so moving it up a row is reasonable, while keeping the position conceptually similar.